### ISSUE TITLE
Regarding dithering in sanear audio renderer

### ISSUE THREAD

**Issue by aproce** [alexmarsev/sanear#1]
Hi, I wanted to know whether you will be providing support for audio dithering when the output bit depth is 24 bit, in the new audio renderer. It would be really great if you could implement 2 LSB peak to peak TPDF (Triangular Probability Density Function) dithering when the bit depth is reduced from floating point audio to 24 bit or lower. Alternatively, you can also allow usage of VST plugins for dithering. 
Thanks, aproce

**Comment by alexmarsev** [comment]
Yes, I plan to implement ->24bit dither, but only after switching to double precision processing chain (whenever I can, some underlying libraries only accept single precision floats).

VST plugins are trickier, no concrete plans regarding them yet.

**Comment by aproce** [comment]
Ok, thank you. When you implement 24 bit dither, please keep the industry standard TPDF dither (2 LSB) as an option in case you choose to implement some other dither algorithm.

I am really looking forward to trying the new audio renderer. 
Kind regards, aproce.
